pipelines/h3c/rtp/v1/task_scripts/do_EXTRACT_HH.sh

```diff
@@ -12,7 +12,7 @@ fn="${1}"
 
 # make the name of the output file based on the input file
 # we put "HH" immediately after the JD
-fn_out=`basename $fn | sed -E "s/(zen.[0-9]{7}\.[0-9]{5}\.)/\1HH./g"`
+fn_out=$(inject_hh ${fn})
```

Comment:
Can we test this to make sure it works as intended? It looks right by eye, but I've been burned by simple-looking regex's in the past...

Reply:
we may actually want to use `"s/(zen\.[0-9]{7}\.[0-9]{5}\.)/\1HH./g"` instead right now we are matching zen + any-character + 7 digits + . + 5 digits + . 
do we need to escape the '.' in the 'HH.' in the replacement clause? Not like it can be a wildcard right?

Reply:
Matt found a missing escape. Shouldn't make a difference in the end, but now we're a little more explicit.

Reply:
Oh wait, did Paul want me to check the function call, not the regex? 
I did test that with:
```
#!/bin/bash
set -e

src_dir="$(dirname "$0")"
source ${src_dir}/_common.sh

fn=/home/beards/zen.2458098.12345.uvh5

fn_out=$(inject_hh ${fn})

echo "file out is:"
echo $fn_out
```

Reply:
That's exactly what I was looking for, thanks!